src/generic/AhoCorasick.jl

```diff
@@ -0,0 +1,203 @@
+#
+#   FreeAssAhoCorasick.jl : implement bulk divide check for leading terms of free associative Algebra elements
+#   for use e.g. in Groebner Basis computation
+#
+###############################################################################
+
+
+export search, AhoCorasickAutomaton, insert_keyword!, aho_corasick_automaton, AhoCorasickMatch#, Word
+
+import DataStructures.Queue, DataStructures.enqueue!, DataStructures.dequeue!
+
+const Word = Vector{Int}
+
+Markdown.@doc doc"""
+AhoCorasickAutomaton
```

Comment:
```suggestion
    AhoCorasickAutomaton
```
Needs to be indented